src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -50,7 +52,36 @@ public PersonCard(Patient patient, int displayedIndex) {
         phone.setText(patient.getPhone().value);
         address.setText(patient.getAddress().value);
         email.setText(patient.getEmail().value);
-        patient.getTags().forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
+        initTags(patient);
+    }
+    /**
+     * Returns the color style for {@code tagName}'s label.
+     */
+    private String getTagColorStyleFor(String tagName) {
+        int i = 0;
+        if (tagName.equals("pioneer")) {
```

Comment:
Perhaps a switch case will be better here!

You can just return the color string rather than putting them in an array!